root/etc/s6-overlay/s6-rc.d/svc-beets/run

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/with-contenv bash
 # shellcheck shell=bash
 
+if [[ -v DROP_ENABLED ]]; then
```

Comment:
Not sure if an ENV var is better than grepping `beet version` to see if the plugin is enabled.